client/login/magic-login/request-login-email-form.jsx

```diff
@@ -75,6 +75,9 @@ class RequestLoginEmailForm extends React.Component {
 						onDismissClick={ this.onNoticeDismiss }
 						status="is-error" />
 				}
+				<h1 className="magic-login__formheader">
```

Comment:
That being said, I'm happy to change it to be more consistent with everything else.